package.json

```diff
@@ -24,6 +24,7 @@
     "which": "^1.0.9"
   },
   "scripts": {
+    "preinstall": "npm install tsd && tsd update",
```

Comment:
tsd isn't installed global here, so it might not be on the path... keep the ./node_modules/tsd/build/cli.js instead?